mobile/src/lib/background.js

```diff
@@ -0,0 +1,72 @@
+import { getStore } from './store'
+import { initService } from './init'
+import { logException } from './crash-reporter'
+import { loadState } from './localStorage'
+import { startMediaUpload, mediaBackup, endMediaUpload } from '../actions/mediaBackup'
+import { backupAllowed } from './network'
+import { initPolyglot } from '../../../src/lib/I18n'
+
+const hasIosCordovaPlugin = () => {
+  return window.cordova !== undefined && window.cordova.platformId === 'ios' && window.BackgroundFetch !== undefined
+}
+
+export const startBackgroundService = () => {
+  if (hasIosCordovaPlugin()) {
+    startIosBackgroundService()
+  }
+}
+
+export const stopBackgroundService = () => {
+  if (hasIosCordovaPlugin()) {
+    stopIosBackgroundService()
+  }
+}
+
+const startIosBackgroundService = () => {
+  let Fetcher = window.BackgroundFetch
+
+  const fetchCallback = () => {
+    console.log('[js] BackgroundFetch initiated')
+
+    loadState().then(persistedState => {
+      let store = getStore(persistedState)
+      initService(store)
+
+      logException('C\'est moi le Service Background!!!')
```

Comment:
I forgot to write in English.

Since the background service of iOS is rather obscure, we want to leave the log to enable us to validate more easily and allow us to see the frequency of emission.